codeql (#25)
it-at-m/lhm_actions: docs/actions.md

### action-checkout

Action to wrap [GitHub Action Checkout](https://github.com/actions/checkout)

> using current branch with default values and [Path-Filter](https://github.com/dorny/paths-filter). Input is the filter. Output are the filtert paths.

Author: ```suggestion
using current branch with default values.
```

```yaml
- uses: it-at-m/lhm_actions/action-templates/actions/action-checkout@v1.0.0
```

> 2. Calculate Filter

Author: ```suggestion
### action-filter
 [Path-Filter](https://github.com/dorny/paths-filter). Input is the filter. Output are the filtert paths.

1. Calculate Filter
```

- uses: it-at-m/lhm_actions/action-templates/actions/action-checkout
```

### action-filter

> [Path-Filter](https://github.com/dorny/paths-filter). Input is the filter. Output are the filtert paths.

Reviewer: This doesn't fit to the other documentation:
- a sentence what this Action does
- an example with inputs and outputs
if it's just a wrapper around an existing Action you can link to the wrapped Action (instead of the example)

Author: ```suggestion
[Path-Filter](https://github.com/dorny/paths-filter) GitHub Action hat enables conditional execution of workflow steps and jobs, based on the files modified by pull request, on a feature branch, or by the recently pushed commits.
```yml
- uses: it-at-m/lhm_actions/action-templates/actions/action-filter@main
  id: changes
  with:
    filters: |
      src:
        - 'src/**'

  # run only if some file in 'src' folder was changed
- if: steps.changes.outputs.src == 'true'
  run: ...
```
```
